### DISCUSSION TITLE
[Support]: Birdseye cameras are very pixelated / noisy for sharp cameras

### DISCUSSION THREAD

**Discussion by Mincka** [blakeblackshear/frigate#11643] [General Support]
### Describe the problem you are having

I have a few 4k Hikvision cameras that are quite sharp and I use their third stream (1080p) for detection in Frigate.

Their thumbnails and live are good in all the views, except in Birdseye where the images become very pixelated / noisy.
This issue is a lot less visible, or not visible at all for other cameras that are less sharp.

I tried to change:
1) The birdseye resolution from 720p to 1080p
2) The quality from 8 to 31

It did not change anything regarding the pixelated effect.

Would it be possible to get a smoother image for these cameras?

Birdseye:
![image](https://github.com/blakeblackshear/frigate/assets/6184289/782ec18f-f39a-4f44-8986-d6a6fdaecbfc)

Elsewhere  in the GUI:
![image](https://github.com/blakeblackshear/frigate/assets/6184289/ccd4d5c2-3f44-479d-883c-10edf96b416e)
![image](https://github.com/blakeblackshear/frigate/assets/6184289/a5bf8d28-f95e-4c88-910f-740cb35f195c)
![image](https://github.com/blakeblackshear/frigate/assets/6184289/3ff02ba5-a85c-4066-91a0-ead2a5331da4)

### Version

dev-a763ae3

### Frigate config file

```yaml
birdseye:
  enabled: true
  width: 1920
  height: 1080  
  quality: 8
  restream: true
  mode: continuous
```

### Relevant log output

```shell
Not relevant
```

### FFprobe output from your camera

```shell
Not relevant
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Hikvision DS-2CD2186G-I

**Comment by NickM-27** [comment]
Quality is inverse. 1 is highest quality.

**Reply by Mincka** [reply]
Indeed, my bad. Also tested with 1 and this "too crisp" effect is still present.
